pages/glossary/instrument.md

```diff
@@ -9,5 +9,12 @@ An [HTML element][] that when [activated][] allows an end-user to achieve an obj
 
 **Note**: Any rule that uses this definition must provide an unambiguous description of the objective the instrument is used to achieve.
 
+#### Background
```

Comment:
This is going to result in a duplicate ID, unfortunately. We don't have a fix for this yet. Maybe change this to:

```suggestion
#### Background About Instrument
```